apps/nextjs-website/src/app/layout.tsx

```diff
@@ -34,6 +35,10 @@ function makeCookieScript(dataDomainScript?: string) {
   `;
 }
 
+export const metadata: Metadata = {
+  metadataBase: new URL('https://developer.pagopa.it'),
```

Comment:
I honestly don't know what happen adding this line of code. My concern is: what happen when we deploy on dev environment, where the base URL is different?